sonarpedia.json

```diff
@@ -3,7 +3,7 @@
   "languages": [
     "JAVA"
   ],
-  "latest-update": "2021-04-26T08:13:38.034417Z",
+  "latest-update": "2021-05-20T20:48:15.570465835Z",
```

Comment:
Should we really commit this, despite the fact that we only partially updated the metadata?

Reply:
indeed. removed